pages/game.js

```diff
@@ -23,22 +31,52 @@ class Game extends React.Component {
     };
   }
 
+  sleep = time => new Promise(resolve => setTimeout(resolve, time));
+  poll = (p, time) => p().then(this.sleep(time).then(() => this.poll(p, time)));
+
+  getClock = async () => {
+    const { game } = this.props;
+    const { publicRuntimeConfig } = getConfig();
+    const url = `${publicRuntimeConfig.BASEURL}`;
+
+    const result = await fetch(
+      `${url}/api/boxscore/${game.basicGameData.gameId}`,
+    );
+    const g = await result.json();
+    return g;
+  };
+
+  componentDidMount() {
```

Comment:
this is nice, but my instincts tell me this logic should happen in `Scoreboard` component. Its already taking in the `game` data that this is operating on and handling the display. What do you think?

Reply:
My plan for this is in here:
https://github.com/mfountoulakis/hoopify/blob/gameText/pages/game.js
on line 77: 

We could poll for the endpoint every minute, once the game becomes active, we present the scoreboard, if not, we display:

```{`The ${teamNames[hTeam.triCode]} take on the ${
            teamNames[vTeam.triCode]
          } @ ${arena.name}. Game
            starts at ${startTimeEastern}`}```

We just don't want a situation where we'd have to poll the same endpoint twice. We could move the polling to the scoreboard if you want to move all the presentation there. How do you think it should look if we moved it?

Reply:
ah word, gotcha. that does makes sense, lets move forward with this route.